cypress/integration/focus-trap-demo.spec.js

```diff
@@ -52,7 +52,7 @@ describe('<FocusTrap> component', () => {
 
       // trap can be deactivated and return focus to lastly focused element before trap is activated
       cy.get('@testRoot')
-        .findByRole('button', { name: 'deactivate trap' })
+        .findByRole('button', { name: "deactivate trap for 'defaults' demo" })
```

Comment:
Same thing here... Can we use a `^deactivate trap` regex somehow?

Reply:
Same here. I've updated this to `.findByRole('button', { name: /^deactivate trap/ })` and it works beautifully.